Packs/NutanixHypervisor/Integrations/NutanixHypervisor/NutanixHypervisor_description.md

```diff
@@ -0,0 +1,31 @@
+## Nutanix
+
+### Available Alert Type IDs, Impact Types, and Entity Types
+Alert Policies contains the list of all possible alerts in the system,
+and their ID, impact type, and entity type. In order to check the optional values for these entities:
+
+1. Go to your Nutanix UI and navigate to the *Alerts* screen. (Can be found under the second tab on the top left corner, see image below)
+2. Navigate to the Alert Policies tab on the left sidebar.
+You should see the following screen:
+![Nutanix Alert Policy](../../doc_files/Alert_Policy.png)
```

Comment:
hmm not sure this will work in the detailed description

Reply:
You mean the image?
It used to work and it indeed doesn't work.
Is there an explanation somewhere how to add images to description file?
@Itay4